NOTES.ZOS

```diff
@@ -0,0 +1,73 @@
+
+ NOTES FOR THE Z/OS PLATFORM
+ ===========================
+
+ First of all note that OpenSSL is currently NOT supported on z/OS.
+
+ However, if you want to build OpenSSL on z/OS anyway, here are a few
+ hints on how to do that. Though these hints are intended to be helpful,
+ they might not work for you at all, depending on your build environment.
+
+ Building OpenSSL 1.1.1
```

Comment:
Er.....  why do we want this on master?  If anything, I suggest these kind of "configuration packages" get added to our wiki.

For the suggested changes in 10-main.conf and 00-base-templates.conf, simply add them there, and add `c99.sh` as well...  although I would suggest `util/` rather than `tools/` since it's an internal utility.

Reply:
Yes, its a 1.1.1 thing. Addind it to the wiki is probably a better solution than backporting this kind of ducomentation/configuration.